examples/repo_example/hashed_bin_delegation.py

```diff
@@ -162,6 +162,7 @@ def find_hash_bin(path: str) -> str:
 #              10-17                       10 11 12 13 14 15 16 17
 #              ...                         ...
 #              f8-ff                       f8 f9 fa fb fc fd fe ff
+assert isinstance(roles["bins"].signed.delegations.roles, dict)
```

Comment:
Because of `mypy` I have to assert that `roles["bins"].signed.delegations.roles` is None as a few lines later I have an assignment. 
Will change the assert to verify that `roles["bins"].signed.delegations.roles`  is `None as if it's not none it can only be a dict.